src/bar/BarChartWithErrors.ts

```diff
@@ -0,0 +1,64 @@
+import {Bar} from 'vue-chartjs'
+import ErrorBarsPlugin from 'chartjs-plugin-error-bars'
+import Component, {mixins} from "vue-class-component";
+import {Prop, Watch} from "vue-property-decorator";
+import {BarChartData} from "./utils";
+
+@Component
+export default class BarChartWithErrors extends mixins(Bar) {
```

Comment:
This class name confused me a bit because of it similarity to BarchartWithFilters, our main component. I wondered if we were offering an 'error' version and a 'filters' version of out component, until I realised this is the equivalent of chartjsBar in HINT. That isn't a great name either tbh, but it does make it clear it's a different sort of beast from our top level component. This is more like 'chartJsConfiguration' or something?

Reply:
I don't think it is really a different sort of beast. It's just a simpler component, but could be used as a standalone component, as in here: https://github.com/reside-ic/vue-charts/pull/1/files#diff-f8f922a46a5a2248996d54e6f4b535d8

I mean rally BarChartWithFilters should be BarChartWithErrorsAndFilters but that's obviously way too verbose! Any other suggestions for terser names?

Reply:
That's true. I guess I wasn't thinking of them as being on the same level, partly because this takes a much more chartJs-y data format. What if we just renamed BarChartWithErrors to BarChart? We could eventually make it configurable whether it uses the errors plugin or not so then you could have BarchartWithFilters but without Errors if you wanted... (though there woudn't be much point using it on its own without errors, rather than just vue-chartjs Bar)